apps/site-app/src/containers/Accounts/Redeem/index.tsx

```diff
@@ -137,31 +136,25 @@ export const Redeem = () => {
     }
 
     // Wait on the given hash
-    const transferValues = {
-      link: (
-        <a target="_blank" href={`https://ropsten.etherscan.io/tx/${response.data.hash}`}> here </a>),
-    };
-    setTransferMessage(translations.step3);
+    setTransferMessage({
+      ...translations.step3,
+      values: {
+        link: (
+          <a target="_blank" href={`https://ropsten.etherscan.io/tx/${response.data.hash}`}> here </a>),
+      }
+    });
     setPercentComplete(0.5);
-    setTransferValues(transferValues);
 
     const tx = await contract.provider.getTransaction(response.data.hash);
     // Wait at least 2 confirmations
     tx.wait(2);
-    const receipt = await contract.provider.getTransactionReceipt(
```

Comment:
not sure why this would return null.  However, this info doesn't really seem to have any use, so removed it